package.json

```diff
@@ -5,7 +5,7 @@
     "backbone": "~1.3.2",
     "coffee-script": "1.6.1",
     "edx-pattern-library": "0.13.0",
-    "edx-ui-toolkit": "~1.1.0",
+    "edx-ui-toolkit": "git+https://github.com/edx/edx-ui-toolkit.git#dc6d6fe3e03ee656fbe2b8e9445fd2fd30cf9781",
```

Comment:
I assume this will be updated to the new release of the ui-toolkit before this is submitted.